resources/[system]/chat/html/App.js

```diff
@@ -6,7 +6,8 @@ window.APP = {
       style: CONFIG.style,
       showInput: false,
       showWindow: false,
-      suggestions: [],
+      subtrahendSuggestions: [],
```

Comment:
let's keep this one called `suggestions`

Reply:
suggestions are now a computed property from the subtrahent minus the minuend (which are proper math terms).
I could not think of a good replacement for suggestions. `suggestedSuggestions` sounds shit.

then the index.html with the binding on :suggestions prop would have to change. If this keeps to be named suggestions.